lib/api/v3/work_packages/work_package_sql_representer.rb

```diff
@@ -55,8 +55,9 @@ class WorkPackageSqlRepresenter
                  representation: ->(*) { "'WorkPackage'" }
 
         property :id
-
         property :subject
+        property :start_date
+        property :due_date
```

Comment:
```suggestion
        property :startDate, column_name: :start_date
        property :dueDate, column_name: :due_date
```

To align with the regular work package representer properties which are camelCased, this should be changed

Reply:
DONE: 4d3735203774a5990a3d394fb35a9e19739030f0